submissions/EmmanuelUdoh/HTMLTASK/login.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Kimaj: for Electronics, Phones, Computers, Fashion </title>
+        <link href="style.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+       <h1>Kimaj</h1>
+    
+       <b><p> 
+             Kimaj is a market place that is into the sales of electronic gadgets phones, computers, clothings for both men and women. kids are not left out here.
+             We offer the best to our customers round the clock.<br>
+          </p>
+
+          <p>
+               Kimaj! "we do business right". 
+         </p>
+
+          <p> 
+              Do well to sign up.
+         </p>
+
+        <h2>Create An Account</h2>
+        <hr>
+        <form method="POST" action="login.php">
+            <fieldset>
+                <div class="container">
+                    <div>
+                        <label for="firstname">
+                            <b>First Name</b></label><br>
+                         <input type="text" size="65" placeholder="  Enter First Name" name="" />
+                         <br><br>
+                    </div>
+                   
+
+                    <div>
+                        <label for="lastname">
+                            <b>Last Name</b></label><br>
+                         <input type="text" size="65" placeholder="  Enter Last Name" name="" />
+                         <br><br>
+                    </div>
+                    
+                   
+                    <div>
+                        <label for="email">
+                            <b> Email</b></label><br>
+                         <input type="text" size="65" placeholder="  Enter Email" name="" />
+                         <br><br>
+                    </div>
+                    
+                   
+                    <div>
+                        <label for="phone number">
+                           <b>Phone</b></label><br>
+                          <input type="text" size="65" placeholder="  Enter phone number" name="" />
+                          <br><br>
+                    </div>
+
+                    <!--for password-->
+                    <div>
+                        <label for="password">
+                            <b>Password</b></label>  &emsp;  &emsp; &emsp; &emsp; &emsp; <input type="checkbox" onclick="myFunction()">Show Password
```

Comment:
Where is myFunction() ?

Reply:
![image](https://user-images.githubusercontent.com/11851535/68593553-04c75800-0496-11ea-8dfd-11426a173835.png)
Show password seems to be floating away...please review